### ISSUE TITLE
It does not add/toggle class if it is rendered by f-each

### ISSUE THREAD

**Issue by evgeniyPP** [fluorjs/fluor#17]
```html
<template f-each="todo in todos">
    <li class="todo">
        <span f-text="todo.title"></span>
    </li>
</template>

<script type="fluor">
    setup("todos", [{ title: "Hello Flour", completed: false }]);
    on("click", ".todo", addClass("completed"));
</script>
```

The code above does not work.

**Comment by madx** [comment]
Hi, thanks for reporting this!
This is actually a lack of information in the docs, an maybe a counter-intuitive part of how Fluor works, but when you use `f-each`, each child item gets it's own [molecule](https://fluorjs.github.io/#guide-molecules) so the click event handler that you setup with `on` cannot access the `<li>` directly.

One way to sove this is to move the event handler inside your `<li>` element:

```js
<template f-each="todo in todos">
    <li class="todo">
        <span f-text="todo.title"></span>
      <script type="fluor">
        on("click", $root, addClass("completed"));
    </script>
    </li>
</template>

<script type="fluor">
    setup("todos", [{ title: "Hello Fluor", completed: false }]);
</script>
```

You can see this code live at https://codepen.io/madx/pen/RwrNVLL

I might change the way it works in the future, but right now that's how it does

**Comment by madx** [comment]
@evgeniyPP I published fluor@0.6.1 that solves this issue! Events are now handled by the root node of the molecule and delegated to the matching selectors.

I made a CodePen with your example and you can see it is now working: https://codepen.io/madx/pen/gOPbEKy
